tutorials/btp-hana-xs-app-create/btp-hana-xs-app-create.md

```diff
@@ -0,0 +1,349 @@
+
+    The application descriptors are mandatory and describe the framework in which an SAP HANA XS classic application runs.
+
+    The `.xsapp` file indicates the root point in the package hierarchy where content is to be served to client requests; the `.xsaccess` file defines who has access to the exposed content and how.
+
+
+    > By default, the project-creation Wizard creates the application descriptors automatically.  
+     If they're not present, you see a 404 error message in the Web Browser when you call the XS JavaScript service. In this case, you need to create the application descriptors manually. For more information, see the [SAP HANA Developer Guide for SAP HANA Studio](https://help.sap.com/viewer/product/SAP_HANA_PLATFORM/2.0.06/en-US?task=discover_task).
+
+7.  Open the context menu for the new file and select **Team** > **Activate**.
+
+  The activate operation publishes your work and creates the corresponding catalog objects. You can now test it.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Test your application)]
+
+Check if your application is working and if the `Hello, World!` message is displayed.
+
+1.  In the SAP BTP cockpit, choose **SAP HANA / SAP ASE** > **Databases & Schemas**.
+
+2.  Select the relevant SAP HANA tenant database in the list.
+
+3. Open the **SAP HANA Web-based Development Workbench** link under **Development Tools**.
+
+4. When the web editor opens, copy the first part of the URL up to and including the **"com"**.
+
+5. To form your target URL, add a **/** followed by the project name.
+
+6. Then add another **/** followed by the name of the file we just added in the previous step.
+
+    **Example**
+
+    **URL that you need to adapt:**
+
+    `<https://mydadb01wfddybp1mp.int.sap.eu2.hana.ondemand.com/sap/hana/ide/>`
+
+    **Project name:** hana-one
+
+    **File name:** MyFirstSourceFile.xsjs
+
+    **Target URL:**
+
+    `<https://mydadb01wfddybp1mp.int.sap.eu2.hana.ondemand.com/hana-one/MyFirstSourceFile.xsjs>`
+
+7. Place the target URL in your web browser address bar and press **`ENTER`** to see the response.
+
+### Results
+
+The following text should be displayed:
+
+**`Hello, World !`**
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 6: ](Retrieve data from SAP HANA)]
+
+To extract data from the database, you use your JavaScript code to open a connection to the database and then prepare and run an SQL statement. The results are added to the **`Hello, World !`** response. You use the following SQL statement to extract data from the database:
+
+`select * from DUMMY`
+
+The SQL statement returns one row with one field called **DUMMY**, whose value is **X**.
+
+1.  In the **Repositories** view in the **SAP HANA Development** perspective in Eclipse, open the `MyFirstSourceFile.xsjs` file in the embedded JavaScript editor.
+
+2.  In the `MyFirstSourceFile.xsjs` file, replace your existing code with the following code:
+
+    ```
```

Comment:
Format as JavaScript code

Reply:
Which formatting should I use? Please advise.